driver_scripts/setup_MOC_basins.py

```diff
@@ -19,6 +19,8 @@ def spcall(args): #{{{
 
 options, args = parser.parse_args()
 
+groupNameMOC = 'MOCRegionsGroup'
+groupNameBasins = 'BasinRegionsGroup'
```

Comment:
What about OceanBasinsGroup?
I wanted to leave Regions ingroupNameMOC because at some point we will also have MOCTransectsGroup.
Also, this is the only place where we define specific groups for now.
Another 2 regions that should fall in the OceanBasinsGroup are the GlobalOcean_* features: any suggestion in whether this should be done by hand or within the setup_ocean_basins driver?